pkg/web/api.go

```diff
@@ -82,15 +84,158 @@ func (api *API) RegisterFilters(filters ...Filter) {
 	api.Filters = append(api.Filters, filters...)
```

Comment:
should check that name is valid:
1. not empty
2. does not already exist
3. does not contain : delimiter because we will use it as a special symbol